src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResource.kt

```diff
@@ -92,17 +116,25 @@ class AuditResource(
   )
   fun findPage(
     pageable: Pageable = Pageable.unpaged(),
-    @RequestParam who: String? = null,
-    @RequestParam what: String? = null,
-  ): Page<AuditDto> = auditService.findPage(pageable, who, what)
+    @RequestBody auditFilterDto: AuditFilterDto
+  ): Page<AuditDto> {
+    log.info("About to start search for audit events")
```

Comment:
Good point. I have defined the second log level as debug, because I know this could be subjective topic. Some projects mandate having an intro log statement on the first line of a controller/service/repo method. I have also seen the pattern in some HMPPS controllers. But, I am all for consistency and continuous improvement of our services and code base. So thanks!